cmake/GenerateC.cmake

```diff
@@ -0,0 +1,8 @@
+
+function(generate_c_string_array FILE VARNAME CONTENT)
```

Comment:
What's this for?

Reply:
Left over from when I was including a list of shader include paths for the bgfx shader compiler.
Doesn't seem like we need it anymore so I'll remove it